server/validation.js

```diff
@@ -337,6 +337,13 @@ const ParticipantQuerySchema = yup.object().shape({
   ),
 });
 
+const ParticipantEditSchema = yup.object().shape({
```

Comment:
Without specifying no additional fields, a malicious actor could slip in another field to be edited. See my comments above. Adding `noUnknown` should fix this.
```suggestion
const ParticipantEditSchema = yup.object().noUnknown('Unknown field in form').shape({
```